### PR TITLE
Feature/#31

### PR DESCRIPTION
**Pull request by yousuke-murayama**
注文履歴画面を作成しました。

### CODE REVIEW

File: src/main/java/com/example/controller/ShowOrderHistoryController.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package com.example.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShowOrderHistoryController.java:7]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.MissingJavadocTypeCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing a Javadoc comment.

File: src/main/java/com/example/controller/ShowOrderHistoryController.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+@RequestMapping("/show-order-history")
+public class ShowOrderHistoryController {
+
+	@GetMapping("")
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShowOrderHistoryController.java:11]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ShowOrderHistoryController.java:11]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def modifier' has incorrect indentation level 8, expected level should be 2.

File: src/main/java/com/example/controller/ShowOrderHistoryController.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+	public String showOrderHistory() {
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShowOrderHistoryController.java:12]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/controller/ShowOrderHistoryController.java [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+		return "ec/order_history";
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShowOrderHistoryController.java:13]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ShowOrderHistoryController.java:13]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/ShowOrderHistoryController.java [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+	}
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShowOrderHistoryController.java:14]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ShowOrderHistoryController.java:14]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def rcurly' has incorrect indentation level 8, expected level should be 2.

File: src/main/java/com/example/controller/ShowOrderHistoryController.java [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShowOrderHistoryController.java:15]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

**Review by uedamiku — APPROVED** [review]

**Pull request merged**